CHANGELOG.md

```diff
@@ -13,11 +13,13 @@ Package-specific changes not released in any SDK will be added here just before
 - Updated `react-native-view-shot` from `3.3.0` to `3.4.0`. ([#19405](https://github.com/expo/expo/pull/19405) by [@douglowder](https://github.com/douglowder))
 - Updated `react-native-webview` from `11.23.0` to `11.23.1`. ([#19375](https://github.com/expo/expo/pull/19375) by [@aleqsio](https://github.com/aleqsio))
 - Updated `react-native-gesture-handler` from `2.5.0` to `2.7.0`. ([#19362](https://github.com/expo/expo/pull/19362) by [@tsapeta](https://github.com/tsapeta))
+<<<<<<< HEAD
```

Comment:
```suggestion
```
😅

Reply:
Yeah, saw that too, but you were just a sec too quick 😢